packages/sn-filter-pane/src/ext/property-panel/data/data-panel/index.ts

```diff
@@ -117,11 +117,10 @@ export default function getDataPanelItems(env: IEnv) {
     autoConfirm: {
       ref: 'autoConfirm',
       component: 'checkbox',
-      translation: 'properties.filterpane.autoConfirm',
+      translation: 'properties.filterpane.showSelectionsToolbar',
```

Comment:
I guess this new string should be added in the sense-client? Do that as soon as possible, so we can get the translations ready soonish

Reply:
It's in already (merged on Friday).